### PR TITLE
Build tools libraries as shared + build alt plaforms

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
There still seems to be many static libraries that are shipped with this. is this intentional?

**Comment by hmaarrfk** [comment]
would you like my help maintaining this? I'm trying to update our (internal) graphics stack

File: recipe/bld.bat [hunk 1]
```diff
@@ -3,6 +3,8 @@ cd build
 cmake -GNinja ^
   -DCMAKE_BUILD_TYPE=Release ^
   -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+  -DSPIRV_TOOLS_LIBRARY_TYPE=SHARED ^
+  -DSPIRV_TOOLS_BUILD_STATIC=OFF ^
```
Comment (hmaarrfk) [inline recipe/bld.bat]:
```suggestion
```

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: conda-forge.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```
Comment (isuruf) [inline conda-forge.yml]: No need

**Review by isuruf — APPROVED** [review]
Yes, I can use help in maintaining.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Pull request merged**